plugins/bufbuild/connect-go/v1.0.0/buf.plugin.yaml

```diff
@@ -1,8 +1,8 @@
 version: v1
 name: buf.build/bufbuild/connect-go
 plugin_version: v1.0.0
-source_url: github.com/bufbuild/connect-go
-description: A better gRPC.
+source_url: https://github.com/bufbuild/connect-go
+description: Type-safe client and server code generator for connect-go.
```

Comment:
lgtm :tada:

If there's space for more text, I'd love to add "Compatible with the gRPC, gRPC-Web, and Connect RPC protocols."